config/software/setuptools.rb

```diff
@@ -16,16 +16,17 @@
 #
 
 name "setuptools"
-default_version "0.7.7"
+default_version "28.3.0"
```

Comment:
Last is `28.6.1`, we might want it since there is a Suse fix: https://github.com/pypa/setuptools/blob/master/CHANGES.rst#v2840